### PR TITLE
Workspaceapi

### PR DESCRIPTION
**Pull request by bunsenstraat**

### CODE REVIEW

File: packages/api/src/lib/file-system/file-panel/api.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ export interface IFilePanel {
         customAction: (cmd: customAction) => void
     } & StatusEvents
     methods: {
-        getCurrentWorkspace(): { name: string, isLocalhost: boolean }
+        getCurrentWorkspace(): { name: string, isLocalhost: boolean, workSpacesPath: string }
```
Comment (yann300) [inline packages/api/src/lib/file-system/file-panel/api.ts:12]: could you explain why you need this params?

Comment (bunsenstraat) [reply packages/api/src/lib/file-system/file-panel/api.ts:12]: https://github.com/ethereum/remix-project/pull/990/files#diff-06867df237e883f50474f3dec9c1cd760265d4e8f1d20376f3807e6683c6582dR45

Comment (bunsenstraat) [reply packages/api/src/lib/file-system/file-panel/api.ts:12]: I know I shouldn't be public but what else ...

Comment (yann300) [reply packages/api/src/lib/file-system/file-panel/api.ts:12]: ah ok I see, in that case do you think we could return the absolute path of the workspace. like:
`{ name: string, isLocalhost: boolean, absolutePath: string }` .
Would this still work with dGit?

**Review by yann300 — APPROVED** [review]

**Pull request merged**